frontend/src/scenes/experiments/Metrics/ExperimentMetricModal.tsx

```diff
@@ -37,15 +37,15 @@ export function ExperimentMetricModal({
 
     const handleSetMetric = useCallback(
         (newMetric: ExperimentMetric): void => {
-            if (!metricIdx) {
+            if (metricIdx == null) {
```

Comment:
complete nitpick: if we want to be strict with types, `metricIdx` is of type `number | null,` and by doing a loose equality check, we are including `undefined`. But I can live with this 😉 .